src/ext/oblivc/obliv_bits.c

```diff
@@ -1842,6 +681,7 @@ void feedOblivInputs(OblivInputs* spec, size_t count, int party)
     { __obliv_c__##ot rv; \
       OblivInputs spec; \
       setupObliv##tname(&spec,&rv,v); \
+      spec.src_f = v; \
```

Comment:
Given that `spec.src_f` is unioned with `spec.src`, won't this overwrite the src variable with bogus data for all types that _aren't_ float? If I am understanding correctly, this should be done in `setupOblivFloat()` to avoid that problem.